sdk/avalon_sdk/tcf_connector.toml

```diff
@@ -79,3 +79,7 @@ gas_price = "100"
 [fabric]
 fabric_network_file = "sdk/avalon_sdk/fabric/network.json"
 channel_name = "mychannel"
+
+[WorkerConfig]
```

Comment:
I assume these values should match with that present in https://github.com/hyperledger/avalon/blob/master/config/tcs_config.toml. Please add a comment in this file.

Reply:
Done.